messages/de/app.php

```diff
@@ -1,153 +1,165 @@
-    'Discord servers tagged with <strong>{tag}</strong>' => 'Discord Server mit dem Stichwort <strong>{tag}</strong>',
-    'Don\'t forget to check "Set this link to never expire"' => 'Vergiss nicht den Link auf "\'Niemals ablaufen\'" zu setzen',
+    'Discord servers tagged with <strong>{tag}</strong>' => 'Discord Server mit dem Stichwort <strong>{tag}<strong>',
+    'Don\'t forget to check "Set this link to never expire"' => 'Vergiss nicht, die Einladung auf "Niemals ablaufen" zu setzen',
     'Edit' => 'Bearbeiten',
     'Edit Server' => 'Server bearbeiten',
-    'Enter keyword(s)' => 'Gib Stichworte ein',
-    'Failed to Sync' => 'Fehler beim Synchronisieren',
+    'Enter keyword(s)' => 'Stichworte eingeben',
+    'Failed to Sync' => 'Synchronisierung fehlgeschlagen',
     'Games' => 'Spiele',
     'Gaming' => 'Gaming',
-    'Get link' => 'URL erhalten',
-    'Go to Dashboard' => 'Gehe zum Dashboard',
+    'Get link' => 'Link erhalten',
+    'Go to Dashboard' => 'Zum Dashboard',
     'Go Back' => 'Zurück',
     'Go Home' => 'Zur Startseite',
+    'Guidelines' => 'Richtlinien',
     'Home' => 'Startseite',
-    'If you don\'t add a bot, you can NOT present various informations about your server, create an invitation link or bump up your servers with commands' => 'Wenn du den Bot nicht zu deinem Server hinzufügst, können verschiedene Informationen auf der Serverseite (z.B. die Anzahl an Servermitgliedern) nicht angezeigt, Einladungslinks nicht erstellt oder Bump-Befehle nicht verwendet werden',
-    'If your server isn\'t in the above list or dropdown, sync with Discord manually' => 'Wenn dein Server nicht in der Liste oben oder im Dropdown-Menü angezeigt wird, synchronisiere bitte manuell mit Discord',
-    'Invite is not set' => 'Einladungslink nicht festgelegt',
-    'You can now write reviews and add ratings for servers you are joining.<br>Let\'s recommend your favorite servers to other people!' => 'Du kannst jetzt für Server, denen du beigetreten bist, Rezensionen schreiben und Bewertungen hinzufügen. <br>Empfehle anderen Leuten deine Lieblingsserver!',
+    'If you don\'t add a bot, you can NOT present various informations about your server, create an invitation link or bump up your servers with commands' => 'Falls du den Bot nicht hinzufügst, können verschiedene Informationen über deine. Server nichr dargestellt werden und mit einem Befehl ein Einladungslink generiert oder dein Server gebumpt werden.',
+    'If your server isn\'t in the above list or dropdown, sync with Discord manually' => 'Falls dein Server nicht in der obigen Liste oder im Dropdown vorhanden ist, synchronisiere manuell mit Discord',
+    'Invite is not set' => 'Einladungslink ist nicht gesetzt',
+    'You can now write reviews and add ratings for servers you are joining.<br>Let\'s recommend your favorite servers to other people!' => 'Du kannst nun Rezensionen schreiben und Bewertungen zu Servern dene du beitrittst hinzufügen.',
```

Comment:
Suggestion: "**Du kannst nun zu Servern, denen du betrittst, Rezensionen schreiben und Bewertungen hinzufügen. Empfehle anderen Menschen deine Lieblingsserver!**"

And the last part was missing ^

Reply:
Oh I just noticed that's basically what was there before, lol. So yeah, I'd suggest to stay with that.